core/modules/system/tests/system.test

```diff
@@ -2360,6 +2360,27 @@ class SystemValidTokenTest extends BackdropWebTestCase {
 }
 
 /**
+ * Tests backdrop_set_message() and related functions.
+ */
+class BackdropSetMessageTest extends BackdropWebTestCase {
+
```

Comment:
Let's add the testing profile here:
```
protected $profile = 'testing';
```

That'll make for a lighter testing installation for this trivial test.